projects/plugins/boost/app/assets/src/js/features/speed-score/pop-out/pop-out.tsx

```diff
@@ -77,6 +78,10 @@ function PopOut( { scoreChange }: Props ) {
 
 	const hideAlert = () => setClose( true );
 
+	if ( hasScoreChanged ) {
+		recordBoostEvent( 'speed_score_alert_shown', {} );
```

Comment:
Since we're showing the alert, we might as well include the scores. I think it makes sense to include whether the scores went up or down as well. Having the alert show by itself wouldn't help us. We could probably see a trend or something if we had more data.

What do you think?

Reply:
Good idea. I added those details. In testing I saw that it was sending the analytics twice, which apparently is React's development rendering the popup twice. It shouldn't happen in production. A way around this is to use useEffect().